README.md

```diff
@@ -6,28 +6,124 @@ See NUbook for a more detailed documentation of this project: <https://nubook.ne
 
 ![Field Example](./docs/outputs/goals_example.gif)
 
-# Requirements
-+ [Blender](https://www.blender.org/download/) (2.79)
+# Prerequisite
+
+Before starting, download and install [Blender 2.79](https://www.blender.org/download/previous-versions/).
 
 # Usage
-## Generating field UV map
-A UV map for the field can be generated by running [`generate_uv.py`](./field_uv/generate_uv.py). This will use the configuration listed in [`scene_config.py`](./pbr/config/scene_config.py).
 
-## Building the scene
-To generate the scene with default field UV map, the `pbr` script should be run through the Blender Python API. That is, `blender --python pbr.py` while in the pbr directory. This will create a scene, rendering a ball, goals and a field depending on the HDR metadata.
+## Setting Up
+
+- Clone this repo
+- Change into the cloned `NUpbr` directory, then into `pbr`
+- Install dependencies by running `ensure_dependencies.py` _using the Python binary installed with Blender_. You may have to run it twice: first to install Pip, and then to install the dependencies.
+- Download the [resources.zip](http://10.1.0.223:8080/share.cgi?ssid=05GE7Tx) file from the NUbots NAS and copy the `resources` directory from it into the `NUpbr` root directory. Note: to access the download, you need to be in the NUbots lab and connected to the local network.
+
+## Building a Scene
+
+To generate a scene with default field UV map, do the following:
+
+- Change into the `pbr` directory
+- Run `pbr.py` using Blender's Python API: `blender --python pbr.py`
+- To run the script without the Blender UI, use: `blender -b --python pbr.py`
```

Comment:
This will put the output folders in the NUpbr root directory rather than in the `pbr` directory. I think this feels slightly cleaner to me
```suggestion
- Run `pbr.py` using Blender's Python API: `blender --python pbr/pbr.py`
- To run the script without the Blender UI, use: `blender -b --python pbr/pbr.py`
```